common/library/module_utils/input_validation/common_utils/en_us_validation_msg.py

```diff
@@ -18,6 +18,8 @@
 These messages are used to provide user-friendly error messages during configuration validation.
 """
 
+MISSING_CLUSTER_NAME_MSG = "Cluster name is mandatory for all kubernetes roles."
+CLUSTER_NAME_OVERLAP_MSG = "Cluster name '{0}' is used in both service and non-service Kubernetes roles, which is not allowed."
```

Comment:
The cluster name '{0}' cannot be shared between service and compute Kubernetes roles.